package.json

```diff
@@ -10,7 +10,7 @@
     "watch": "NODE_ENV=development webpack --watch --progress",
     "css": "WEBPACK_CHILD=css npm run build",
     "tx": "tx push --source && tx pull --all && node transifex.js",
-    "lint": "eslint client/src && lint-sass",
+    "lint": "eslint client/src && stylelint client/src/**/*.scss",
     "lint-js": "eslint client/src",
     "lint-js-fix": "eslint client/src --fix",
     "lint-sass": "stylelint **/client/src/**/*.scss"
```

Comment:
It looks like I skipped `yarn` before `lint-sass`. It should run custom `lint-sass` command. Or, probably, we should remove this command `"lint-sass": "stylelint **/client/src/**/*.scss"` at all.

Reply:
Updated